package.json

```diff
@@ -65,9 +66,11 @@
     "standard-version": "^7.0.1"
   },
   "dependencies": {
+    "axios": "^0.20.0",
     "lodash": "^4.17.15",
     "request": "^2.88.0",
```

Comment:
We'll need a follow-up TODO ticket for removing `request` so that the lib isn't getting too messy with multiple XHR libs

Reply:
see: https://rewardops.atlassian.net/browse/MX-1063